### PR TITLE
Fix first Python demo so it works without further modifications

### PR DESCRIPTION
**Pull request by simonw**
You can't do `async with` ... outside of an `async def` function.

### CODE REVIEW

**Comment by simonw** [comment]
After applying this fix you can save the code as `/tmp/demo.py` and configure it in the inspector like this:

CleanShot 2024-11-25 at 08 45 09@2x

**Comment by simonw** [comment]
The inline script dependencies trick here also means you don't even have to install `mcp` separately. See: https://simonwillison.net/2024/Nov/24/follow-theirs/

File: README.md [hunk 1]
```diff
@@ -76,6 +76,11 @@ Connections between clients and servers are established through transports like
 MCP servers follow a decorator approach to register handlers for MCP primitives like resources, prompts, and tools. The goal is to provide a simple interface for exposing capabilities to LLM clients.
 
 ```python
+# /// script
+# dependencies = [
+#   "mcp"
+# ]
+# ///
```
Comment (simonw) [inline README.md:83]: The inline script dependencies trick here also means you don't even have to install `mcp` separately. See: https://simonwillison.net/2024/Nov/24/follow-theirs/

**Review by dsp-ant — APPROVED** [review]
Thank you so much @simonw !

File: README.md (hunk 1 above)
Comment (dsp-ant) [reply README.md:83]: TIL. Thank you

**Pull request merged**